lib/mindwendel/brainstormings/brainstorming.ex

```diff
@@ -3,13 +3,19 @@ defmodule Mindwendel.Brainstormings.Brainstorming do
 
   import Ecto.Changeset
   alias Mindwendel.Brainstormings.Idea
+  alias Mindwendel.Brainstormings.IdeaLabel
   alias Mindwendel.Accounts.User
   alias Mindwendel.Accounts.BrainstormingUser
 
   schema "brainstormings" do
     field :name, :string
     field :admin_url_id, :binary_id
+    # ToDo: Please change the previous line to this one
```

Comment:
not sure what this means?

Reply:
I refined the comment. It is meant to be a reminder for small refactoring in the future.